R/board_datatxt.R

```diff
@@ -9,16 +9,17 @@ datatxt_refresh_index <- function(board) {
   if (is.null(board$url)) stop("Invalid 'url' in '", board$name, "' board.")
 
   index_file <- "data.txt"
+  index_url <- file_path_null(board$url, board$subpath, index_file)
+
+  index_file_get <- file_path_null(board$subpath, "data.txt")
```

Comment:
```suggestion
  index_file_get <- file_path_null(board$subpath, index_file)
```
?

Reply:
Thanks @hadley, I have updated and commited your suggested change.